internal/provider-config/config.go

```diff
@@ -27,9 +28,10 @@ var mutex sync.RWMutex
 type IncusProviderRemoteConfig struct {
 	Name         string
 	Address      string
-	Port         string
+	Protocol     string
+	Auth_Type    string
```

Comment:
Please follow our naming convention (style and use complete words): `AuthenticationType`